### ISSUE TITLE
FP on rule TrailingCommas when line is ending with ;

### ISSUE THREAD

**Issue by racodond** [iwarapter/sonar-puppet#269]
Should not raise an issue when last parameter is ending with `;` as on the example below:

```
class abc {
  sonarqube::plugin {
    'sonar-css-plugin':
      version => $css;
   'sonar-java-plugin':
      abc => $abc,
      version => $java;
  }
}
```

because the following is not really nice

```
version => $java,;
```
